src/promo/stories/donationDialog.jsx

```diff
@@ -44,7 +44,7 @@ class ExampleDialog extends React.Component {
           </div>
 
           <Button colour='primary' onClick={this.handleOpen}>Open Dialog</Button>
-          <DonationDialog href='https://donate.theconversation.com' open={this.state.open} onClose={this.handleClose} donateText='Donate now'>
+          <DonationDialog open={this.state.open} onClose={this.handleClose} donateText='Donate now'>
```

Comment:
Stylistic + documentation: for all "promos" stories that handle now have an `onClick` action, I've added a Storybook `action` addon callback on them so I can see that they execute an action whenever they are clicked:

```jsx
import { action } from '@storybook/addon-actions'

// ...
<DonationBanner onClick={action('clicked')}>
```

Reply:
@tiagoamaro Good point, will do.